helm/preview/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
All the `preview` folder should go in the already existing `ci/helm/` folder, no need to create a new `helm` folder at the root.